image-layout.md

```diff
@@ -148,6 +148,7 @@ The [image index](image-index.md) is a multi-descriptor entry point.
 This index provides an established path (`/index.json`) to have an entry point for an image-layout and to discover auxiliary descriptors.
 
 No semantic restriction is given for the "org.opencontainers.ref.name" annotation of descriptors.
+The value of "org.opencontainers.ref.name" annotation is a string which MUST NOT include characters outside of the set of "A" to "Z", "a" to "z", "0" to "9", the hyphen `-`, the dot `.`, and the underscore `_`.
```

Comment:
The original regexp also has some more constraints on corner-cases which are allowed by your wording:
 * empty string
 * arbitrarily-long string (> 128 chars)
 * symbol-only string (eg. `..`)
 * string starting with a symbol (eg. `.hidden`)